.github/workflows/integration-tests.yml

```diff
@@ -136,7 +136,14 @@ jobs:
         uses: docker://bitnami/openldap
         with:
           entrypoint: /bin/bash
-          args: -c "(/opt/bitnami/openldap/bin/ldapwhoami -H ldap://ldap:3389 -D cn=admin,dc=symfony,dc=com -w symfony||sleep 5) && /opt/bitnami/openldap/bin/ldapadd -H ldap://ldap:3389 -D cn=admin,dc=symfony,dc=com -w symfony -f src/Symfony/Component/Ldap/Tests/Fixtures/data/fixtures.ldif && /opt/bitnami/openldap/bin/ldapdelete -H ldap://ldap:3389 -D cn=admin,dc=symfony,dc=com -w symfony cn=a,ou=users,dc=symfony,dc=com"
+          args: -c "(/opt/bitnami/openldap/bin/ldapwhoami -H ldap://ldap:3389 -D
+            cn=admin,dc=symfony,dc=com -w symfony||sleep 5) &&
+            /opt/bitnami/openldap/bin/ldapadd -H ldap://ldap:3389 -D
+            cn=admin,dc=symfony,dc=com -w symfony -f
+            src/Symfony/Component/Ldap/Tests/Fixtures/data/fixtures.ldif &&
+            /opt/bitnami/openldap/bin/ldapdelete -H ldap://ldap:3389 -D
+            cn=admin,dc=symfony,dc=com -w symfony
+            cn=a,ou=users,dc=symfony,dc=com"
```

Comment:
Why are you reformatting this ?

Reply:
For better readability

Reply:
and why doing that in a commit saying "use pinned dependencies" without any insight from the maintainers about whether it is even appropriate to change this ?
Thus, I'm not convinced about the readability argument here, where the YAML newlines inside the quoted string make it less clear what is actually run.

Reply:
I understand, I can undo this part if there is consensus to merge this issue. Please let me know.